index.js

```diff
@@ -32,7 +34,51 @@ class TaskList {
     description.value = "";
     priority.value = "low";
     this.changeTaskPriorityIndicator();
-    console.log(this.tasks);
+    this.displayTasks();
+  }
+  displayTasks() {
+    const tasksContainer = document.getElementById("tasks-list");
+    tasksContainer.innerHTML = "";
+    const priorityColors = {
+      low: "lightgreen",
+      medium: "yellow",
+      high: "darkred",
+    };
+
+    this.tasks.forEach((task) => {
+      const taskItem = document.createElement("div");
+      taskItem.classList.add("task-item");
+      taskItem.id = task.id;
+      if (task.completed) taskItem.classList.add("completed");
+      taskItem.style.backgroundColor = priorityColors[task.priority] || "white";
+
+      taskItem.innerHTML = `
+      <div class="task-body">
+      <div class="task-info">
+      <h3>${task.title}</h3>
+      <p>${task.description}</p>
+      </div>
+      </div>
+      <div class="task-btns">
+      <button class="task-item-complete-btn" id="task-item-complete-btn">${
+        task.completed ? incompleteIcon() : completeIcon()
+      }</button>
+      <button class="task-item-edit-btn">${editIcon()}</button>
+      <button class="task-item-delete-btn">${deleteIcon()}</button>
+      </div>
+    `;
```

Comment:
I dont prefer to use this approach, Please use something like `document.createElement("div")` and `document.createElement("h3")`... and not `.innerHTML = xxxx`